packages/ts/src/rules/newExpressions.test.ts

```diff
@@ -0,0 +1,80 @@
+import rule from "./newExpressions.js";
+import { ruleTester } from "./ruleTester.js";
+
+ruleTester.describe(rule, {
+	invalid: [
+		{
+			code: `
+new MyClass();
+`,
+			snapshot: `
+new MyClass();
+~~~~~~~~~~~~~~
```

Comment:
Have the report range only be on `new`. Constructors may have long params.

```suggestion
new MyClass();
~~~
```